shotover-proxy/tests/kafka_int_tests/test_cases.rs

```diff
@@ -169,6 +172,138 @@ pub async fn produce_consume_partitions1(
     }
 }
 
+pub async fn produce_consume_partitions1_kafka_node_goes_down(
+    driver: KafkaDriver,
+    docker_compose: &DockerCompose,
+    connection_builder: &KafkaConnectionBuilder,
+    topic_name: &str,
+) {
+    if driver.is_cpp() {
```

Comment:
I recall we can use the feature flag of driver types to control if tests cases are run. Can we use that approach here?

Reply:
Say if we used feature flags here and set this test case to `#[cfg(not(feature = "kafka-cpp-driver-tests")]`, then if we enable all feature flags (which is how we run our tests in CI) then this test case would not run at all.

We dont have a feature flag for the java driver since its always available.

So the check really needs to be done at runtime rather than compile time.

In general, the purpose of feature flags are to enable compiling on platforms that dont support certain drivers or to reduce compile times, not to exclude test cases in certain conditions.